testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -517,3 +525,17 @@ def pillar_get(key, minion)
   raise 'File injection failed' unless return_code.zero?
   $server.run("chmod 644 " + dest)
 end
+
+When(/^I migrate "([^"]*)" from salt-minion to venv-salt-minion$/) do |host|
+  node = get_target(host)
+  system_name = node.full_hostname
+  migrate = "salt #{system_name} state.apply util.mgr_switch_to_venv_minion"
+  $server.run(migrate, check_errors: true, verbose: true)
+end
+
+When(/^I clean up "([^"]*)" everything salt-minion related$/) do |host|
```

Comment:
I'm not sure I understand this step - we already have a step to clean up salt stuff left behind, what does this do exactly?

Reply:
Ah, maybe the step could have a better name. It is directly from the SUMA documentation regarding cleaning up a Salt minion after a migration. See https://documentation.suse.com/suma/4.3/en/suse-manager/client-configuration/contact-methods-saltbundle.html#_switching_from_salt_minion_to_salt_bundle.
![image](https://user-images.githubusercontent.com/12104291/230061044-5199be99-9b64-4721-9c7b-d886ed6d5156.png)

Should I rename the step to make it clearer?

Reply:
Yes, please! I'd rename it something like 
```suggestion
When(/^I purge salt-minion on "([^"]*)" after migration$/) do |host|
```
just to make it clearer it's related to the migration and not the same cleanup we do when making a host unreachable